cmd/applyconfig/applyconfig.go

```diff
@@ -119,14 +124,14 @@ func (c commandExecutor) runAndCheck(cmd *exec.Cmd, action string) ([]byte, erro
 
 	if output, err = cmd.Output(); err != nil {
 		if exitError, ok := err.(*exec.ExitError); ok {
-			fmt.Printf("ERROR: %s: failed to %s\n%s\n", pretty, action, exitError.Stderr)
+			logrus.Errorf("%s: failed to '%s' '%s'", pretty, action, exitError.Stderr)
```

Comment:
I do not like the removing of separator between the action and stderr (which can be quite long)

Reply:
I got the impression that  `\n` is not line break in logrus.
https://github.com/sirupsen/logrus/issues/608

Reply:
If that is true, then the output will be quite unreadable (the Stderr will very likely contain multiple lines) and I wonder what do we actually gain by switching to logrus...

Reply:
The censor is the reason we switched.
Let me try to get an example.

Reply:
line breaker seems fine ...

```
$  ./applyconfig --config-dir /tmp/test_ac
INFO[0000] Applying standard config in directory: /tmp/test_ac 
ERRO[0000] oc apply -f /tmp/test_ac/test.yaml --dry-run: failed to 'apply' 'error: Missing or incomplete configuration info.  Please login or point to an existing, complete config file:

  1. Via the command-line flag --config
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
' 

```

So created https://github.com/openshift/ci-tools/pull/238
:laughing: